backend/app/api/endpoints/adapter/flows.py

```
@@ -0,0 +1,257 @@
+        flow_id=flow_id,
+        user_id=current_user.id,
+    )
+
+
+# ========== Webhook Trigger Endpoint ==========
+
+
+@router.post("/webhook/{webhook_token}", response_model=FlowExecutionInDB)
+def trigger_flow_webhook(
+    webhook_token: str,
+    payload: Dict[str, Any] = {},
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)